### ISSUE TITLE
Null menuRef when hot reload React

### ISSUE THREAD

**Issue by danielpaz6** [szhsin/react-menu#1097]
React/React-dom version: 18.2.0 
React-Menu version: 4.0.0

## Describe the bug
I was changing something not related to react menu in my website, and after the hot reloader started to kick in and refresh the page, I received the following error:
```
Cannot read properties of null (reading 'contains')
TypeError: Cannot read properties of null (reading 'contains')
    at http://localhost/static/js/bundle.js:87537:30
```

That points out to the file MenuList.js at line 318:
```
if (!menuRef.current.contains(document.activeElement)) {
```
<img width="711" alt="image" src="https://github.com/szhsin/react-menu/assets/42693974/7d0ae532-9c02-48ff-9997-5f50c4b071b4">

I'd suggest to ask first if MenuCurrent exists, or use optional chaining operator

**Comment by szhsin** [comment]
Thanks @danielpaz6 for reporting the issue. Will be fixing it in the next release.

**Comment by szhsin** [comment]
Fixed by [v4.1.0](https://github.com/szhsin/react-menu/releases/tag/v4.1.0)
